litellm/litellm_core_utils/litellm_logging.py

```diff
@@ -121,6 +123,7 @@
 berrispendLogger = None
 supabaseClient = None
 liteDebuggerClient = None
+opikLogger = None
```

Comment:
we're moving away from adding loggers like this to litellm_logging.py - please make use of the CustomLogger() class. See `langsmith.py` on how we implement custom loggers